src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java

```diff
@@ -57,6 +61,22 @@
     public Field makeField(SQLExpr expr, String alias, String tableAlias) throws SqlParseException {
         Field field = makeFieldImpl(expr, alias, tableAlias);
 
+        if (expr.getParent() != null && expr.getParent() instanceof SQLSelectItem
```

Comment:
Can we add a comment here on what exactly is the workaround is.

Reply:
Sure, will do!